content/docs/addins/concepts/addin-sso/index.md

```diff
@@ -14,7 +14,7 @@ SKY add-ins support a single-sign-on (SSO) mechanism that can be used to correla
 
 Within the <a href="https://github.com/blackbaud/sky-addin-client" target="_new">Add-in Client JavaScript library</a>, the `AddinClient` class provides a `getAuthtoken` function for getting a short-lived "user identity token" from the host application. This token is a signed value that is issued to the SKY API application and represents the Blackbaud user's identity.
 
-The general flow is that when an add-in is initiated, it can request a user identity token from the host page using the `getAuthtoken` function.  The host will in turn request a user identity token from the SKY API OAuth 2.0 service.  The user identity token (a <a href="https://jwt.io">JWT</a>), will be addressed to the SKY API application, and will contain the Blackbaud user's unique identifier.  The OAuth service will return the token to the host, and the host will pass the token to the add-in.  The add-in can then pass the user identity token (along with the environment ID provided as part of the initial context when the add-in is instantiated) to its own backend, where it can be validated and used to look up a user in the add-in's native system.  If a user mapping exists, then the add-in can consider the user logged in, and immediately display content to the user.  If no user mapping exists, the add-in can prompt the user to login to the add-in's native system.  Once the user's identity in the native system is known, the add-in can persist the user mapping so that on subsequent loads the user doesn't have to log in again (even across devices).
+The general flow is that when an add-in is initiated, it can request a user identity token from the host page using the `getAuthtoken` function.  The host will in turn request a user identity token from the SKY API OAuth 2.0 service.  The user identity token (a <a href="https://jwt.io">JWT</a>), will be addressed to the SKY API application and will contain the Blackbaud user's unique identifier.  The OAuth service will return the token to the host, and the host will pass the token to the add-in.  The add-in can then pass the user identity token (along with the environment ID provided as part of the initial context when the add-in is instantiated) to its own backend, where it can be validated and used to look up a user in the add-in's native system.  If a user mapping exists, then the add-in can consider the user logged in and immediately display content to the user.  If no user mapping exists, the add-in can prompt the user to login to the add-in's native system.  Once the user's identity in the native system is known, the add-in can persist the user mapping so that on subsequent loads the user doesn't have to log in again (even across devices).
```

Comment:
Only change was that I removed the second comma from:

If a user mapping exists, then the add-in can consider the user logged in, and immediately display content to the user.